schema/tables/screenlock.yml

```diff
@@ -1,7 +1,8 @@
 name: screenlock
-description: >- 
-  macOS screenlock status
-notes: >- 
-  - Only fetches results for osquery's current logged-in user context. If osquery is running as root, this table will return no results. 
-  
-  - The user must also have recently logged in in order to show the latest setting. 
+platforms:
+  - darwin
+  - chrome
+description: >-
+  Returns if the screen locks automatically and the time, in seconds, it takes until the screen is locked automatically while idle.
+notes: >-
+  - For macOS, this only fetches results for osquery's current logged-in user context. The user must also have recently logged in.
```

Comment:
Curious about why this was removed:
```
If osquery is running as root, this table will return no results. 
```
Seemed useful to me.

Reply:
Oh snap! Let me add that back in, that's only for macOS, should we put it in the description or in the notes?

Reply:
I'm ok with either. Though given that the table is not working on macOS we should put it on the description.